### PR TITLE
Bug 1123564 - initial work of device deck, r=rex

### PR DESCRIPTION
**Pull request by evelynhung**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[evelynhung evelynhung](https://github.com/evelynhung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=429092e9e362)

File: tv_apps/device-deck/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Devices</title>
+
+    <link rel="stylesheet" type="text/css" href="style/devices.css">
```
Comment (rexboy7) [inline tv_apps/device-deck/index.html]: `</head>`

Comment (rexboy7) [inline tv_apps/device-deck/index.html]: And we may add some l10n library
```
    <!-- Shared L10n libraries -->
    <script src="shared/js/l10n.js"></script>
    <script src="shared/js/l10n_date.js"></script>

    <!-- L10n properties -->
    <meta name="defaultLanguage" content="en-US">
    <meta name="availableLanguages" content="en-US">
    <link rel="localization" href="locales/devices-deck.{locale}.properties"/>
    <link rel="localization" href="shared/locales/date/date.{locale}.properties"/>
```

Comment (evelynhung) [inline tv_apps/device-deck/index.html]: I removed them on purpose since we don't have any strings now. Would it cause any problem in build time?

**Comment by try-server-hook** [comment]
[evelynhung evelynhung](https://github.com/evelynhung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7738233c761e)

**Comment by try-server-hook** [comment]
[evelynhung evelynhung](https://github.com/evelynhung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7f58c5302d15)

File: tv_apps/device-deck/index.html (hunk 1 above)
Comment (evelynhung) [inline tv_apps/device-deck/index.html]: alright, I add them back.

**Pull request merged**